examples/grpc-service-reactor/src/main/java/example/armeria/grpc/reactor/HelloServiceImpl.java

```diff
@@ -0,0 +1,108 @@
+package example.armeria.grpc.reactor;
+
+import com.linecorp.armeria.server.ServiceRequestContext;
+import example.armeria.grpc.reactor.Hello.HelloReply;
+import example.armeria.grpc.reactor.Hello.HelloRequest;
+import example.armeria.grpc.reactor.ReactorHelloServiceGrpc.HelloServiceImplBase;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.core.scheduler.Schedulers;
+
+import java.time.Duration;
+import java.util.stream.Collectors;
+
+public class HelloServiceImpl extends HelloServiceImplBase {
+
+    /**
+     * Sends a {@link HelloReply} immediately when receiving a request.
+     */
+    @Override
+    public Mono<HelloReply> hello(Mono<HelloRequest> request) {
+        return request.map(it -> buildReply(toMessage(it.getName())));
+    }
+
+    /**
+     * Sends a {@link HelloReply} 3 seconds after receiving a request.
+     */
+    @Override
+    public Mono<HelloReply> lazyHello(Mono<HelloRequest> request) {
+        // You can use the event loop for scheduling a task.
+        return request
+            .delayElement(Duration.ofSeconds(3), Schedulers.fromExecutor(ServiceRequestContext.current().contextAwareEventLoop()))
```

Comment:
Maybe better breaking the line somewhere?

Reply:
done